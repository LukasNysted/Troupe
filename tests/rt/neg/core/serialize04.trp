(* negative test checking the interaction between sandboxing and serialization; 
   resumption konts that we receive from the sandbox must not be serializable

*)


let fun f() = _debug (1); sleep (2000); _debug (2)
    val (2, resume) = sandbox (1000, f)
in  save ("serialize04", resume)
end
