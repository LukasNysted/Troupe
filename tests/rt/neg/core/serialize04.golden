2021-06-12T11:46:58.515Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
1
[31mRuntime error in thread e49a2325-6bb6-4bff-b9ea-db9894c1f801@{}%{}[39m
[31m>> Unserializable object: <sandboxkont>@{}%{}[39m
